Compilation errors on Ubuntu 14.04 - 32 bit

When I try to build on a 32 bit Ubuntu 14.04, I get the following error:

```
$ make install
[  1%] Generating C++ from XSD files...
/usr/bin/xsd: 2: /usr/bin/xsd: Syntax error: ")" unexpected
make[2]: *** [Malmo/src/Mission.h] Error 2
make[1]: *** [Malmo/src/CMakeFiles/Malmo.dir/all] Error 2
make: *** [all] Error 2
```

--- a/doc/build_linux.md
+++ b/doc/build_linux.md
@@ -65,9 +65,9 @@
 
 7. On Debian 7 and Ubuntu 14.04 only: Install CodeSynthesis XSD:
     1. `wget http://www.codesynthesis.com/download/xsd/4.0/linux-gnu/x86_64/xsd_4.0.0-1_amd64.deb`
-     - (Use http://codesynthesis.com/download/xsd/4.0/linux-gnu/i686/xsd_4.0.0-1_i386.deb if installing on a 32 bit machine)
+     - Use `http://codesynthesis.com/download/xsd/4.0/linux-gnu/i686/xsd_4.0.0-1_i386.deb` if installing on a 32 bit machine
     2. `sudo dpkg -i --force-all xsd_4.0.0-1_amd64.deb`  
-     - (`sudo dpkg -i --force-all xsd_4.0.0-1_i386.deb` on a 32 bit machine)
+     - Run `sudo dpkg -i --force-all xsd_4.0.0-1_i386.deb` on a 32 bit machine
     3. `sudo apt-get install -f`  
        This step is needed because we require xsd version 4.0.  
        (When mono-devel is updated, you will need to manually remove then reinstall xsd as above, because of the package conflicts.)

--- a/doc/build_linux.md
+++ b/doc/build_linux.md
@@ -65,7 +65,9 @@
 
 7. On Debian 7 and Ubuntu 14.04 only: Install CodeSynthesis XSD:
     1. `wget http://www.codesynthesis.com/download/xsd/4.0/linux-gnu/x86_64/xsd_4.0.0-1_amd64.deb`
+     - (Use http://codesynthesis.com/download/xsd/4.0/linux-gnu/i686/xsd_4.0.0-1_i386.deb if installing on a 32 bit machine)
     2. `sudo dpkg -i --force-all xsd_4.0.0-1_amd64.deb`  
+     - (`sudo dpkg -i --force-all xsd_4.0.0-1_i386.deb` on a 32 bit machine)
     3. `sudo apt-get install -f`  
        This step is needed because we require xsd version 4.0.  
        (When mono-devel is updated, you will need to manually remove then reinstall xsd as above, because of the package conflicts.)

